src/daos/docker/daos/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Seagate Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build an image of radosgw + DAOS
+#
+
+# Pull base image
+ARG BASE_DISTRO=daos-rocky:latest
+FROM $BASE_DISTRO
+LABEL maintainer="seagate-daos@seagate.com"
+
+# needed to create certs, git, ccache
+RUN sudo dnf install openssl git ccache jq net-tools iproute -y
+RUN sudo dnf update -y
+
+# copy the daos yml files for a local run
+COPY docker_daos_server.yml /opt/daos/etc/daos_server.yml
+COPY docker_daos_agent.yml /opt/daos/etc/daos_agent.yml
+COPY docker_daos_control.yml /opt/daos/etc/daos_control.yml
+ENV DAOS_PATH=/opt/daos
+ENV SOURCE_DATE_EPOCH=946684800
+RUN sudo bash -c 'echo "vm.nr_hugepages = 513" >> /etc/sysctl.conf'
```

Comment:
513?

Reply:
something else is using 1 hugepage, causes DAOS to fail